core/src/main/java/com/microsoft/applicationinsights/internal/quickpulse/DefaultQuickPulseDataFetcher.java

```diff
@@ -50,13 +50,13 @@ public DefaultQuickPulseDataFetcher(ArrayBlockingQueue<HttpRequest> sendQueue, T
         sdkVersion = getCurrentSdkVersion();
         StringBuilder sb = new StringBuilder();
 
-        // FIXME (trask) what about azure functions consumption plan where role name not available yet?
+        // FIXME (heya) what about azure functions consumption plan where role name not available yet?
```

Comment:
it will get lazily set later. It's already handled in LazyConfigurationAccessor.setWebsiteSiteName.

Reply:
the prefix in this class doesn't get updated at that time tho